deploy/tasks.py

```diff
@@ -0,0 +1,171 @@
+from invoke import task
```

Comment:
In light of what we've discussed yesterday during the infra FG call, please create an issue to migrate this file to JavaScript.

Reply:
Done. Ticket filed [here](https://github.com/medic/cht-core/issues/8551).